compiler/rustc_lint_defs/src/lib.rs

```diff
@@ -886,20 +892,23 @@ macro_rules! declare_tool_lint {
     (
         $(#[$attr:meta])* $vis:vis $tool:ident ::$NAME:ident, $Level: ident, $desc: expr
         $(, @feature_gate = $gate:expr;)?
+        $(, [loadbearing: $loadbearing: literal])?
     ) => (
         $crate::declare_tool_lint!{$(#[$attr])* $vis $tool::$NAME, $Level, $desc, false $(, @feature_gate = $gate;)?}
     );
     (
         $(#[$attr:meta])* $vis:vis $tool:ident ::$NAME:ident, $Level:ident, $desc:expr,
         report_in_external_macro: $rep:expr
         $(, @feature_gate = $gate:expr;)?
+        $(, [loadbearing: $loadbearing: literal])?
```

Comment:
Could you also keep the same syntax as the other cases: either `@something: true` or `something: true` without the brackets?